Description: add efl as internal dep for eldbus
Author: Romain Naour <romain.naour@openwide.fr>
Applied-Upstream: 22d2d11d2289f76a9e75d38fcf01b7ced4cbb9e7
Last-Update: 2016-01-16
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/configure.ac
+++ b/configure.ac
@@ -2992,6 +2992,7 @@
 
 ### Checks for libraries
 EFL_INTERNAL_DEPEND_PKG([ELDBUS], [ecore])
+EFL_INTERNAL_DEPEND_PKG([ELDBUS], [efl])
 EFL_INTERNAL_DEPEND_PKG([ELDBUS], [eo])
 EFL_INTERNAL_DEPEND_PKG([ELDBUS], [eina])
 
